glean-core/src/metrics/mod.rs

```diff
@@ -26,6 +27,7 @@ use crate::Glean;
 pub use self::boolean::BooleanMetric;
 pub use self::counter::CounterMetric;
 pub use self::datetime::DatetimeMetric;
+pub use self::experiment::ExperimentMetric; // FIXME: should this be crate-only? But then how to run tests? Using Glean.set*?
```

Comment:
Yeah, in general it sounds sensible to make this private. Maybe it is enough to test the public API on this?

Reply:
I followed your suggestion, however I have another problem with `RecordedExperimentData`. See the new comment.